### ISSUE TITLE
Multiprocessing on moods_dna.py

### ISSUE THREAD

**Issue by vmesel** [jhkorhonen/MOODS#13]
As I'm running MOODS to search for multiple motifs on 20304 sequences, I would like to see things running quicker. Is there any way to activate multiprocessing on moods_dna.py ?

**Comment by jhkorhonen** [comment]
No, sorry, you'll currently have to run parallel instances of moods_dna.py manually if you want to use multiple cores.

However, first make sure that you are using the `--batch` option. This will have a dramatic effect on performance with lots of sequences. (See also what `--help` say about the `--bg` option.)

**Comment by vmesel** [comment]
**--batch** made it run faster but I still need more velocity on the process. Running one motif on each core should solve the problem on multiple motifs, but I will need to adapt all my code! No future plans on parallelizing this pipeline?

**Comment by jhkorhonen** [comment]
Sorry, I will not have time to do that in the near future.

It's likely more efficient to parallelise in terms of sequences, given what MOODS does. I would just suggest splitting your sequence set into _M_ equal-size sets, where _M_ is the number of cores, and calling moods_dna.py in parallel for each of the _M_ sequence sets, using the full motif set each time.
